aten/src/ATen/native/quantized/cpu/qmul.cpp

```diff
@@ -98,11 +90,10 @@ class QMulScalar final : public c10::OperatorKernel {
   Tensor operator()(Tensor qa, Scalar b,
                     double scale, int64_t zero_point) {
     TORCH_CHECK(qa.qscheme() == kPerTensorAffine ||
-              qa.qscheme() == kPerTensorSymmetric,
+                qa.qscheme() == kPerTensorSymmetric,
```

Comment:
I'm going to remove kPerTensorSymmetric in backend, so you can remove this check